src/Codeception/Subscriber/Console.php

```diff
@@ -536,7 +536,9 @@ protected function detectWidth()
             }
         } elseif ($this->isWin() && (php_sapi_name() === "cli")) {
             exec('mode con', $output);
-            preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            if (isset($output[4])) {
+                preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            }
             if (!empty($matches[1])) {
```

Comment:
The condition `!empty($matches[1])` should nested in `isset($output[4])` since `$matches` can only exist if `preg_match` statement has been executed.